test/cypress/package-lock.json

```diff
@@ -1955,4 +1955,4 @@
       }
     }
   }
-}
+}
```

Comment:
We can revert this change as it's not related to the PR (looks like it just adds an empty line)